web/debugger.mjs

```diff
@@ -20,6 +20,16 @@ for (const key in OPS) {
   opMap[OPS[key]] = key;
 }
 
+// This is copied from src/display/canvas_dependency_tracker.js
+const EMPTY_BBOX = new Uint32Array(new Uint8Array([255, 255, 0, 0]).buffer)[0];
+const BBoxDecoder = {
```

Comment:
This is copy&pasted in a bunch of places. What's the best locations for it, that is accessible in src/display, in web/pdf_page_view, and in the debugger?

Reply:
If you use it only in web/, you can add it in web/ui_utils.js. If you plan to use it in display then you should add it in display/display_utils.js and do something similar to what we've for `getRGB` (just `grep getRGB`) in order to make it usable in web/.